theme/material/partials/nav.html

```diff
@@ -0,0 +1,57 @@
+<!-- Main navigation -->
+<nav class="md-nav md-nav--primary" data-md-level="0">
+
+    <!-- Site title -->
+    <label class="md-nav__title md-nav__title--site" for="__drawer">
+        <a href="{{ config.site_url | default(nav.homepage.url, true) | url }}"
+           title="{{ config.site_name }}" class="md-nav__button md-logo">
+            {% if config.theme.logo.icon %}
+            <i class="md-icon">{{ config.theme.logo.icon }}</i>
+            {% else %}
+            <img src="{{ config.theme.logo | url }}" width="48" height="48" />
+            {% endif %}
+        </a>
+        {{ config.site_name }}
+        <label for="Versions">Ver:
+            <select id="versions" onchange="window.location.href=this.value;" style="
+		position: inherit;
+		display: inline;
+		min-width: 25%;
+		/* width: 120%; */
+		height: 25px;
+		padding: 5px 15px 5px 5px;
+		color: Black;
+		border: none;
+		background: transparent;
+		outline: none;
+		-webkit-appearance: none;
+		z-index: 99;
+		cursor: pointer;
+		font-size: 12px;
+		margin-bottom: 10px;
+		border: 1px solid black;
+		overflow: auto
+	">
+                <option value="">Versions</option>
```

Comment:
Instead of hard coding the versions, lets populate the options (version numbers) from the config file. And populate the options using liquid templating (jinja). Then we can add the selected state to the specific version number.